### PR TITLE
Merge to master BorrowBookControl 

### PR DESCRIPTION
**Pull request by bcurby**
Changes made as per code style guidelines
Merge from master with BorrowBookControl branch completed
Deconflicted BoorowBookControl within the IDE
Committed all resolved changes
Branch is ready for review

### CODE REVIEW

File: ITC205_BaseLine_Code/src/BorrowBookControl.java [hunk 1]
```diff
@@ -1,117 +1,124 @@
...
+    private library library;
+    private member member;
+    private ControlState state;
+    private Book currentBook;
+    
+    private List<Book> pendingBooks;
+    private List<Loan> completedLoans;
+    
+
+
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/BorrowBookControl.java]: D1 - Remove 1 line of white spaces (should be 2, not 3)

**Review by lukexjohnson — COMMENTED** [review]
Pls remove line  20 (1x whitespace)

File: ITC205_BaseLine_Code/src/BorrowBookControl.java (hunk 1 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/BorrowBookControl.java]: Fixed & commit pushed up

**Comment by lukexjohnson** [comment]
D1 - Extra line removed. Merging to master.

**Pull request merged**